python/client/azure/mgmt/redhatopenshift/v2020_04_30/models/_azure_red_hat_open_shift_client_enums.py

```diff
@@ -14,27 +14,12 @@
 # Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code is regenerated.
```

Comment:
Yes, we would, otherwise, need to have a way to prevent older clients from generating with the current modules. It's possible to implement, but doesn't have much value due to a separate repo for the upstream as you said.